src/StructuredLogger/Construction/Construction.cs

```diff
@@ -512,7 +513,17 @@ public void StatusEventRaised(object sender, BuildStatusEventArgs e)
             {
                 lock (syncLock)
                 {
-                    if (e is ProjectEvaluationStartedEventArgs projectEvaluationStarted)
+                    // If the build was canceled we want to show a message in the build log view.
+                    if (e is BuildCanceledEventArgs buildCanceledEventArgs)

```

Comment:
since BuildCanceled is the most rare message, let's put it at the very end of the if chain, so we don't have to test for it first. 

I try to sort these type tests by most common message first, so that in the common case we do less type comparisons.